src/awscli_login/saml.py

```diff
@@ -146,7 +152,7 @@ def refresh(url: str, cookies: str) -> Tuple[str, List[Role]]:
     except FileNotFoundError:
         raise MissingCookieJar(url)
 
-    soap = saml_login(url, jar)
+    soap = saml_login(url, jar, verify_cert)
```

Comment:
This is being interpreted as `username`. So we just need to use a named argument here to allow `username` to continue to default to `None`.
```suggestion
soap = saml_login(url, jar, verify_cert=verify_cert)
```

Reply:
This should clear the build error caught by MyPy, and fix the bug.

Reply:
@kwessel I've added a suggested fix above in this GitHub pull request that should clear the build error. If you're satisfied with it (or just if you trust me) you can hit the `Commit suggestion` button and it will make the change and re-run the build checks.